contracts/ConversionRates2.sol

```diff
@@ -58,8 +58,8 @@ contract ConversionRates2 is ConversionRates {
         public
         onlyOperator
     {
-        require(xBuy.length == yBuy.length);
-        require(xSell.length == ySell.length);
+        require(xBuy.length + 1 == yBuy.length);
+        require(xSell.length + 1 == ySell.length);
         require(xBuy.length <= MAX_STEPS_IN_FUNCTION);
         require(xSell.length <= MAX_STEPS_IN_FUNCTION);
         require(tokenData[token].listed);
```

Comment:
when saving step functions to storage. squeeze data.
Especially required for the read operations.
(its in line 47 but can't comment on that line...)

the x value max size should be int64. same for bps value.

But must verify this values with SCP. Spyros and victor.

its in line 81, but can't comment on that line